dist/s_clang_scan.diff

```diff
@@ -9,3 +9,4 @@ wiredtiger/src/include/intpack_inline.h Assigned value is garbage or undefined [
 wiredtiger/src/reconcile/rec_row.c Branch condition evaluates to a garbage value [core.uninitialized.Branch]
 wiredtiger/src/schema/schema_create.c Null pointer passed to 1st parameter expecting 'nonnull' [core.NonNullParamChecker]
 wiredtiger/src/utilities/util_dump.c Null pointer passed to 1st parameter expecting 'nonnull' [core.NonNullParamChecker]
+wiredtiger/src/utilities/util_dump.c Null pointer passed to 1st parameter expecting 'nonnull' [core.NonNullParamChecker]
```

Comment:
Missing newline

Reply:
Don't know why clang_analyzer starts to fail.

Reply:
That's OK. I don't object to adding this line, it's just missing the last newline in the file now.